corehq/apps/hqwebapp/utils/bootstrap/status/bootstrap3_to_5_completed.json

```diff
@@ -7,5 +7,15 @@
     "javascript": [
       "js/edit_builds.js"
     ]
+  },
+  "case_search": {
+    "templates": [
+      ".html",
```

Comment:
This is weird, I think this line should be `case_search.html`... But this is generated by migration tool... so I don't know if this is a mistake by migration tool or it should be `.html`?

Reply:
I think this file is just to keep track of what work is left. And it does not seem to track all the other apps the Jenny did so far. So I would not worry to much about it @biyeun can you confirm this?

Reply:
I think Biyeun just added this feature recently, and other apps Jenny migrated is before this PR getting merged. Correct me if I'm wrong. @biyeun